### DISCUSSION TITLE
Set nearest neighbor filtering for image() instead of bilinear?

### DISCUSSION THREAD

**Discussion by lucasw** [emilk/egui#150] [Q&A] [answered]
Is it possible to show scaled up image pixels with sharp edges?

**Comment by emilk** [comment, marked as answer]
No, not with the `egui_glium` and `egui_web` backends.

**Reply by zicklag** [reply]
Would you be open to a pull request to add a filtering option to textures?

I was going to make a pixel sprite converter tool with Egui, but I need crisp pixel images so this is a blocker.

**Reply by zicklag** [reply]
Managed to get a version that works with `egui_glow` for my project, but it's a super quick draft so I need to think if there's a cleaner API for it. For now I just needed it to work, so here it is if anybody is interested: #1636.
